src/OrleansRuntime/Hosting/DefaultSiloServices.cs

```diff
@@ -65,6 +68,8 @@ internal static void AddDefaultServices(IServiceCollection services)
             services.TryAddSingleton<LogConsistencyProviderManager>();
             services.TryAddSingleton<BootstrapProviderManager>();
             services.TryAddSingleton<LoadedProviderTypeLoaders>();
+            services.AddLogging();
+            services.TryAddSingleton<IPEndPoint>(sp => sp.GetService<NodeConfiguration>()?.Endpoint);
```

Comment:
Can we avoid the `IPEndPoint` registration, since it's not a service? The object that needs this being injected can just inject `ILocalSiloDetails`, and then access it from `details.SiloAddress.Endpoint`